src/Illuminate/Database/Eloquent/Factories/Factory.php

```diff
@@ -155,10 +155,10 @@ public static function new($attributes = [])
     /**
      * Get a new factory instance for the given number of models.
      *
-     * @param  int  $count
+     * @param  Closure|int  $count
      * @return static
      */
-    public static function times(int $count)
+    public static function times($count)
```

Comment:
This is a breaking change

Reply:
Just for my own understanding, this is potentially breaking because someone might be overriding this method signature by extending this class correct? It seemed like it should still behave the same, despite no longer doing any scalar coercion, based on how `$this->count` is being used internally.

Reply:
Yes, exactly. The following code:
```php
class Factory
{
    public function times($count)
    {
        return value($count);
    }
}

class Child extends Factory
{
    public function times(int $count)
    {
        return $count;
    }
}
```

Will throw `PHP Fatal error:  Declaration of Child::times(int $count) must be compatible with Factory::times($count) in ...`